lesson7/basic_strings_exercises.md

```diff
@@ -8,15 +8,15 @@ parent: 7 - Strings
 
 ## Looking at the Java reference documentation
 
-A reference is a technical documentation that explain all the capabilities available in a class or in a library.
+A reference is a technical documentation that explains all the capabilities available in a class or in a library.
 This exists also for every Java class.
 
 - Google for `java string reference` to find the reference documentation of the `String` class
-  or visit [](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html).
+  or visit !!! web link does not show at all !!! [](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html).
```

Comment:
the problem seems to be that the [] is empty
i.e. either
[https://docs.oracle.com/javase/9/docs/api/java/lang/String.html]

or sth like
[the official Java String Documentation](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html)

should work (depending on what text the link should have)